client/app/components/routes/select-operator/form/UserOperator.tsx

```diff
@@ -25,8 +29,8 @@ async function getBusinessStructures() {
   );
 }
 
-export async function getUserOperatorFormData(id: number | undefined) {
-  if (!id) return {};
+export async function getUserOperatorFormData(id: number | string | undefined) {
+  if (!id || id === "request-access") return {};
```

Comment:
Nice work on this, I think it will be a lot safer. This is the only thing I'm thinking about for when my pr merges as I am doing the same but `id === 'create'`. I think I will approve this and might just refactor it and do a NaN(Number(id)) or some similar check to simplify 👍

Reply:
Sounds good, Thanks 🙏